includes/Admin/AdminBar.php

```diff
@@ -108,19 +140,22 @@ public function visit_dashboard_menu( $wp_admin_bar ) {
             return;
         }
 
-        $dashboard = dokan_get_navigation_url();
-
-        if ( ! $dashboard ) {
-            return;
+        $menus = $this->get_dokan_admin_menus();
+
+        // Show wp admin menus in admin bar here, if menu URL.
+        foreach ( $menus as $menu ) {
+            if ( ! isset( $menu['href'] ) || empty( $menu['href'] ) ) {
```

Comment:
Only `empty()` check is enough here. It will do the both.
And what if one `$menu['href']` is empty, but other `$menu` has.

So, best-case would be - 
```php
foreach ( $menus as $menu ) {
    if ( ! empty( $menu['href'] ) ) {
        $wp_admin_bar->add_node(
              [
                   'parent' => $menu['parent'],
                   'id'     => $menu['id'],
                   'title'  => $menu['title'],
                   'href'   => $menu['href'],
              ]
         );
   }
}
```